packages/web/src/pages/settings-page/components/desktop/AuthorizedApps/AuthorizedAppListItem.tsx

```diff
@@ -59,7 +61,7 @@ export const AuthorizedAppListItem = (props: AuthorizedAppListItemProps) => {
     <li className={styles.listItem}>
       <span className={styles.listItemImage}>
         {imageUrl ? (
-          <img src={imageUrl} />
+          <PreloadImage src={imageUrl} />
```

Comment:
Not familiar with this one. Wonder how it relates to `DynamicImage`? Maybe the latter is just for profile pics and track/playlist artwork...

Reply:
so dynamic image is meant to transition between a low res image to a high res one. and preload image just does the onLoad trick to render it all at once

Reply:
NEAT (image of Bender with eyes extending)